lib/beaker/host/pswindows/file.rb

```diff
@@ -2,17 +2,29 @@ module PSWindows::File
   include Beaker::CommandFactory
 
   def tmpfile(name)
-    execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
+    execute("echo #{system_temp_path}\\#{name}.%RANDOM%")
   end
 
   def tmpdir(name)
     # generate name
-    tmpdirname = execute("echo C:\\Windows\\Temp\\#{name}.%RANDOM%")
-    # created named dir
+    tmpdirname = tmpfile(name)
+    # create named dir
     execute("md #{tmpdirname}")
     tmpdirname
   end
 
+  # Turns a host file path into one that can be used by BitVise SSH
+  #
+  # @param [String] path Path to be changed
+  #
+  # @return [String] network version of that path
+  def network_path(path)
```

Comment:
Ok - I understand not wanting to extend the scope of this issue... should we file another ticket then?

Since the `scp_to` accepts the host as a parameter, it should be able to canonicalize the path to be compatible with the current scp server in use, based on the host's details as discovered at runtime.  This would be a noop on most platforms, but take affect on Windows where different scp servers are being used.  Otherwise, any acceptance tests making calls to `scp_to` themselves would have to be written with up-front knowledge of which backend they're using (Cygwin vs Bitvise style pathing), which seems problematic,  unnecessary and wrong IMHO.  I feel this should be deferred until run-time.

Rather than use a convention as you've suggested for detecting whether BitVise is running, I would suggest introspecting the host to see if it is running Bitvise and set an internal `ssh_server` or similar value on the host (until a later date when host vs shell can be properly separated).  There are a couple of ways I can think to do this:

```
C:\Users\Administrator>cmd.exe /c sc query BvSshServer | find "RUNNING"
        STATE              : 4  RUNNING
```

OR

```
C:\Users\Administrator>cmd.exe /c netstat -a -b | find "BvSshServer"
 [BvSshServer.exe]
 [BvSshServer.exe]
 [BvSshServer.exe]
```

These mechanisms are not 100% perfect, but I think are probably good enough for now.  Does that make sense?